policyengine_us/variables/gov/states/ky/tax/income/exclusions/pension_income/ky_pension_income_exclusion_exemption_eligible.py

```diff
@@ -0,0 +1,27 @@
+from policyengine_us.model_api import *
+
+
+class ky_pension_income_exclusion_exemption_eligible(Variable):
+    value_type = bool
+    entity = Person
+    label = "KY Pension Income Exclusion Exemption Eligible"
+    definition_period = YEAR
+    defined_for = StateCode.KY
+    reference = "https://revenue.ky.gov/Forms/Schedule%20P%202022.pdf"
+
```

Comment:
This can replace the formula
```suggestion
    adds = ["retired_from_federal_government", "retired_from_ky_government", "receives_railroad_benefits"]
```